horace_core/sqw/PixelData/@PixelDataFileBacked/PixelDataFileBacked.m

```diff
@@ -285,6 +285,25 @@ function saveobj(~)
         end
 
 
+        function [obj,unique_idx] = reset_changed_coord_range(obj,field_name)
```

Comment:
JW suggests to JW that he put this in the Hidden block so incorrect calls are not made